src/store/index.js

```diff
@@ -1,115 +1,123 @@
-import { createStore, applyMiddleware } from 'redux';
+import { configureStore, createSlice } from '@reduxjs/toolkit';
 import thunk from 'redux-thunk';
 import { v4 as uuidv4 } from 'uuid';
 
 import { logActionMiddleware } from './logActionMiddleware';
 
-const initialState = {
+const initialCardsState = {
   items: [],
   readOnly: true,
   isAddModalOpen: false,
-  isChecked: false,
-  isEditing: false,
+  editingCardIds: [],
```

Comment:
I think it's a little difficult logic.
You already have 'isEditing' field in each of your cards. Could you please try to use it everywhere instead of creating this array?